index.html

```diff
@@ -19,17 +19,28 @@
 
           <div id="menuContentContainer">
             <ul id="menuContent">
-              <li><a href="index.html">Home</a></li>
               <li><a href="disciplines.html">Disciplines</a></li>
               <li><a href="./dropzones.html">Dropzones</a></li>
               <li><a href="#">Where to Start</a></li>
-              <li><a href="#">About</a></li>
               <li><a href="./contact.html">Contact</a></li>
             </ul>
           </div>
         </div>
       </nav>
     </header>
+    <div id="bGImage">
```

Comment:
bGimage is also a bad name. Background image for what? Better name would be bannerImage